[2021-06-25 19:54:25 +0200] [26739] [INFO] Goin' Fast @ http://95.217.8.18:8000
INFO:sanic.root:Goin' Fast @ http://95.217.8.18:8000
[2021-06-25 19:54:25 +0200] [26739] [INFO] Starting worker [26739]
INFO:sanic.root:Starting worker [26739]
[2021-06-25 20:21:48 +0200] - (sanic.access)[INFO][146.75.181.62:10633]: GET http://95.217.8.18:8000/index.html  404 672
INFO:sanic.access:
[2021-06-25 20:21:54 +0200] - (sanic.access)[INFO][146.75.181.62:10631]: GET http://95.217.8.18:8000/main.html  404 671
INFO:sanic.access:
[2021-06-25 20:21:59 +0200] - (sanic.access)[INFO][146.75.181.62:10632]: GET http://95.217.8.18:8000/swagger  302 0
INFO:sanic.access:
[2021-06-25 20:22:00 +0200] - (sanic.access)[INFO][146.75.181.62:10632]: GET http://95.217.8.18:8000/swagger/  200 1470
INFO:sanic.access:
[2021-06-25 20:22:00 +0200] - (sanic.access)[INFO][146.75.181.62:10632]: GET http://95.217.8.18:8000/swagger/swagger-ui.css  200 143410
INFO:sanic.access:
[2021-06-25 20:22:00 +0200] - (sanic.access)[INFO][146.75.181.62:10645]: GET http://95.217.8.18:8000/swagger/swagger-ui-bundle.js  200 1079764
INFO:sanic.access:
[2021-06-25 20:22:00 +0200] - (sanic.access)[INFO][146.75.181.62:10646]: GET http://95.217.8.18:8000/swagger/swagger-ui-standalone-preset.js  200 336298
INFO:sanic.access:
[2021-06-25 20:22:22 +0200] - (sanic.access)[INFO][91.233.243.237:57011]: GET http://95.217.8.18:8000/swagger/favicon-16x16.png  200 665
INFO:sanic.access:
[2021-06-25 20:22:22 +0200] - (sanic.access)[INFO][91.233.243.237:57012]: GET http://95.217.8.18:8000/swagger/swagger-config  200 2
INFO:sanic.access:
[2021-06-25 20:22:22 +0200] - (sanic.access)[INFO][91.233.243.237:57010]: GET http://95.217.8.18:8000/swagger/favicon-32x32.png  200 628
INFO:sanic.access:
[2021-06-25 20:22:22 +0200] - (sanic.access)[INFO][91.233.243.237:57012]: GET http://95.217.8.18:8000/swagger/swagger.json  200 1919
INFO:sanic.access:
[2021-06-25 20:22:23 +0200] - (sanic.access)[INFO][18.208.182.121:2238]: GET http://95.217.8.18:8000/swagger/swagger.json  200 1919
INFO:sanic.access:
[2021-06-25 21:51:12 +0200] - (sanic.access)[INFO][174.138.57.82:44474]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-26 00:26:16 +0200] - (sanic.access)[INFO][68.183.5.205:48010]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-26 02:10:57 +0200] - (sanic.access)[INFO][184.105.247.195:44986]: GET http://95.217.8.18/  200 55
INFO:sanic.access:
[2021-06-26 02:51:54 +0200] - (sanic.access)[INFO][183.136.225.14:28115]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-26 02:52:25 +0200] - (sanic.access)[INFO][183.136.225.14:59062]: GET http://95.217.8.18:8000/favicon.ico  404 673
INFO:sanic.access:
[2021-06-26 06:36:00 +0200] - (sanic.access)[INFO][54.183.221.192:60084]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 01:23:48 +0200] - (sanic.access)[INFO][167.248.133.56:49960]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 01:23:48 +0200] - (sanic.access)[INFO][167.248.133.56:46584]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 03:11:21 +0200] - (sanic.access)[INFO][180.149.125.175:33750]: GET http://95.217.8.18:8000/c/  404 664
INFO:sanic.access:
[2021-06-27 04:46:07 +0200] - (sanic.access)[INFO][42.193.45.103:41424]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 08:40:36 +0200] - (sanic.access)[INFO][183.136.225.14:57126]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 08:40:37 +0200] - (sanic.access)[INFO][183.136.225.14:23294]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 08:40:37 +0200] - (sanic.access)[INFO][183.136.225.14:32652]: GET http://95.217.8.18:8000/favicon.ico  404 673
INFO:sanic.access:
[2021-06-27 13:29:41 +0200] - (sanic.access)[INFO][174.138.57.82:33532]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 13:32:24 +0200] - (sanic.access)[INFO][180.149.125.175:48303]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-27 14:19:51 +0200] - (sanic.access)[INFO][64.62.197.32:51534]: GET http://95.217.8.18/  200 55
INFO:sanic.access:
[2021-06-27 23:38:09 +0200] - (sanic.access)[INFO][66.60.16.47:47039]: GET http://95.217.8.18/  200 55
INFO:sanic.access:
[2021-06-28 00:46:17 +0200] - (sanic.access)[INFO][20.98.209.47:57877]: HEAD http:///robots.txt  404 672
INFO:sanic.access:
[2021-06-28 02:20:43 +0200] - (sanic.access)[INFO][192.241.208.247:37216]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-28 13:33:16 +0200] - (sanic.access)[INFO][158.255.150.252:41376]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-28 13:33:22 +0200] - (sanic.access)[INFO][158.255.150.252:41382]: GET http://95.217.8.18:8000/index.html  404 672
INFO:sanic.access:
[2021-06-28 13:33:22 +0200] - (sanic.access)[INFO][158.255.150.252:41382]: GET http://95.217.8.18:8000/favicon.ico  404 673
INFO:sanic.access:
[2021-06-28 13:33:30 +0200] - (sanic.access)[INFO][158.255.150.252:41384]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-28 13:37:37 +0200] - (sanic.access)[INFO][158.255.150.252:41552]: GET http://95.217.8.18:8000/worker  403 43
INFO:sanic.access:
[2021-06-28 13:37:45 +0200] - (sanic.access)[INFO][158.255.150.252:41556]: GET http://95.217.8.18:8000/register  405 695
INFO:sanic.access:
[2021-06-28 13:37:56 +0200] - (sanic.access)[INFO][158.255.150.252:41558]: GET http://95.217.8.18:8000/login  200 48
INFO:sanic.access:
[2021-06-28 13:49:14 +0200] - (sanic.access)[INFO][158.255.150.252:41808]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-28 13:49:16 +0200] - (sanic.access)[INFO][158.255.150.252:41808]: GET http://95.217.8.18:8000/  200 55
INFO:sanic.access:
[2021-06-28 13:49:20 +0200] [26739] [INFO] Stopping worker [26739]
INFO:sanic.root:Stopping worker [26739]
[2021-06-28 13:49:20 +0200] [26739] [INFO] Server Stopped
INFO:sanic.root:Server Stopped
[2021-06-28 13:49:23 +0200] [12272] [INFO] Goin' Fast @ http://95.217.8.18:8000
INFO:sanic.root:Goin' Fast @ http://95.217.8.18:8000
[2021-06-28 13:49:23 +0200] [12272] [INFO] Starting worker [12272]
INFO:sanic.root:Starting worker [12272]
[2021-06-28 13:49:23 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/  200 6119
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/fonts/font-awesome-4.7.0/css/font-awesome.min.css  404 711
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/fonts/iconic/css/material-design-iconic-font.min.css  404 714
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/bootstrap/css/bootstrap.min.css  404 700
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/animate/animate.css  404 688
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/css-hamburgers/hamburgers.min.css  404 702
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/animsition/css/animsition.min.css  404 702
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/select2/select2.min.css  404 692
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.css  404 704
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/css/util.css  404 674
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/css/main.css  404 674
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/jquery/jquery-3.2.1.min.js  404 695
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/animsition/js/animsition.min.js  404 700
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/bootstrap/js/popper.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/vendor/bootstrap/js/bootstrap.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/select2/select2.min.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/vendor/daterangepicker/moment.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.js  404 703
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/countdowntime/countdowntime.js  404 699
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/js/main.js  404 672
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/jquery/jquery-3.2.1.min.js  404 695
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/animsition/js/animsition.min.js  404 700
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/popper.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/bootstrap.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/select2/select2.min.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/moment.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.js  404 703
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/countdowntime/countdowntime.js  404 699
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/js/main.js  404 672
INFO:sanic.access:
[2021-06-28 13:49:24 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/images/icons/favicon.ico  404 686
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/?username=test&password=test  200 6119
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/css/bootstrap.min.css  404 700
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/fonts/iconic/css/material-design-iconic-font.min.css  404 714
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/fonts/font-awesome-4.7.0/css/font-awesome.min.css  404 711
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/vendor/css-hamburgers/hamburgers.min.css  404 702
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/animate/animate.css  404 688
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/animsition/css/animsition.min.css  404 702
INFO:sanic.access:
INFO:sanic.access:
INFO:sanic.access:
INFO:sanic.access:
INFO:sanic.access:
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/select2/select2.min.css  404 692
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.css  404 704
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/css/util.css  404 674
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/css/main.css  404 674
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/vendor/jquery/jquery-3.2.1.min.js  404 695
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/animsition/js/animsition.min.js  404 700
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/popper.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/bootstrap/js/bootstrap.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/select2/select2.min.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/vendor/daterangepicker/moment.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.js  404 703
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/vendor/countdowntime/countdowntime.js  404 699
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/js/main.js  404 672
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/jquery/jquery-3.2.1.min.js  404 695
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/animsition/js/animsition.min.js  404 700
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/popper.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/bootstrap.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/select2/select2.min.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/moment.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.js  404 703
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/countdowntime/countdowntime.js  404 699
INFO:sanic.access:
[2021-06-28 13:49:28 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/js/main.js  404 672
INFO:sanic.access:
[2021-06-28 13:49:32 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/?username=user&password=user  200 6119
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/css/bootstrap.min.css  404 700
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/fonts/font-awesome-4.7.0/css/font-awesome.min.css  404 711
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/fonts/iconic/css/material-design-iconic-font.min.css  404 714
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/vendor/animsition/css/animsition.min.css  404 702
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/animate/animate.css  404 688
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/css-hamburgers/hamburgers.min.css  404 702
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/select2/select2.min.css  404 692
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/css/util.css  404 674
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.css  404 704
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/css/main.css  404 674
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/vendor/animsition/js/animsition.min.js  404 700
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/jquery/jquery-3.2.1.min.js  404 695
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/select2/select2.min.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41820]: GET http://95.217.8.18:8000/vendor/bootstrap/js/popper.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41822]: GET http://95.217.8.18:8000/vendor/bootstrap/js/bootstrap.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41824]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.js  404 703
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/daterangepicker/moment.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41826]: GET http://95.217.8.18:8000/vendor/countdowntime/countdowntime.js  404 699
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/js/main.js  404 672
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41818]: GET http://95.217.8.18:8000/vendor/jquery/jquery-3.2.1.min.js  404 695
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/animsition/js/animsition.min.js  404 700
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/popper.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/bootstrap/js/bootstrap.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/select2/select2.min.js  404 691
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/moment.min.js  404 698
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/daterangepicker/daterangepicker.js  404 703
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/vendor/countdowntime/countdowntime.js  404 699
INFO:sanic.access:
[2021-06-28 13:49:33 +0200] - (sanic.access)[INFO][158.255.150.252:41816]: GET http://95.217.8.18:8000/js/main.js  404 672
INFO:sanic.access:
[2021-06-28 13:50:57 +0200] [12272] [INFO] Stopping worker [12272]
INFO:sanic.root:Stopping worker [12272]
[2021-06-28 13:50:57 +0200] [12272] [INFO] Server Stopped
INFO:sanic.root:Server Stopped
[2021-06-28 13:50:59 +0200] [13231] [INFO] Goin' Fast @ http://95.217.8.18:8000
INFO:sanic.root:Goin' Fast @ http://95.217.8.18:8000
[2021-06-28 13:50:59 +0200] [13231] [INFO] Starting worker [13231]
INFO:sanic.root:Starting worker [13231]
[2021-06-28 13:55:34 +0200] - (sanic.access)[INFO][158.255.150.252:41950]: GET http://95.217.8.18:8000/login  200 48
INFO:sanic.access:
[2021-06-28 14:10:59 +0200] - (sanic.access)[INFO][158.255.150.252:42254]: GET http://95.217.8.18:8000/login?username=test&password=testte  200 48
INFO:sanic.access:
[2021-06-28 14:12:35 +0200] - (sanic.access)[INFO][158.255.150.252:42372]: GET http://95.217.8.18:8000/login?username=asd&password=asd  200 48
INFO:sanic.access:
[2021-06-28 14:14:10 +0200] [13231] [INFO] Stopping worker [13231]
INFO:sanic.root:Stopping worker [13231]
[2021-06-28 14:14:10 +0200] [13231] [INFO] Server Stopped
INFO:sanic.root:Server Stopped
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS, cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS, cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS, cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS, cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
[2021-06-28 14:20:58 +0200] [21265] [INFO] Goin' Fast @ http://95.217.8.18:8000
INFO:sanic.root:Goin' Fast @ http://95.217.8.18:8000
[2021-06-28 14:20:58 +0200] [21265] [INFO] Starting worker [21265]
INFO:sanic.root:Starting worker [21265]
[2021-06-28 14:28:36 +0200] - (sanic.access)[INFO][158.255.150.252:42608]: GET http://95.217.8.18:8000/login?username=user&password=123456  200 45
INFO:sanic.access:
[2021-06-28 14:39:57 +0200] - (sanic.access)[INFO][158.255.150.252:44324]: GET http://95.217.8.18:8000/login?username=user&password=pass  200 31
INFO:sanic.access:
[2021-06-28 14:46:24 +0200] - (sanic.access)[INFO][146.75.181.55:11966]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 14:46:28 +0200] - (sanic.access)[INFO][146.75.181.55:11971]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 14:46:50 +0200] - (sanic.access)[INFO][146.75.181.63:12224]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 14:46:53 +0200] - (sanic.access)[INFO][146.75.181.63:12233]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 14:50:48 +0200] - (sanic.access)[INFO][158.255.150.252:44468]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 14:50:55 +0200] - (sanic.access)[INFO][158.255.150.252:44468]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 15:18:34 +0200] [21265] [INFO] Stopping worker [21265]
INFO:sanic.root:Stopping worker [21265]
[2021-06-28 15:18:34 +0200] [21265] [INFO] Server Stopped
INFO:sanic.root:Server Stopped
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 14, in <module>
    from database.entities import User, Complex, Event, Marker, GeographicLocation
  File "/root/leadersofdigital_splavy_rosseti/database/entities/__init__.py", line 1, in <module>
    from .complex import *
  File "/root/leadersofdigital_splavy_rosseti/database/entities/complex.py", line 3, in <module>
    from database.entities.event import Event
  File "/root/leadersofdigital_splavy_rosseti/database/entities/event.py", line 3, in <module>
    from database.entities.misc import Marker
  File "/root/leadersofdigital_splavy_rosseti/database/entities/misc.py", line 3, in <module>
    from server import app
  File "/root/leadersofdigital_splavy_rosseti/server.py", line 2, in <module>
    from sanic_cors import CORS
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/__init__.py", line 11, in <module>
    from .decorator import cross_origin
  File "/usr/local/lib/python3.7/dist-packages/sanic_cors/decorator.py", line 13, in <module>
    from sanic_plugin_toolkit import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/__init__.py", line 4, in <module>
    from .realm import SanicPluginRealm
  File "/usr/local/lib/python3.7/dist-packages/sanic_plugin_toolkit/realm.py", line 18, in <module>
    from sanic.models.futures import FutureException as SanicFutureException
ModuleNotFoundError: No module named 'sanic.models'
[2021-06-28 15:22:13 +0200] [27888] [INFO] Goin' Fast @ http://95.217.8.18:8000
INFO:sanic.root:Goin' Fast @ http://95.217.8.18:8000
[2021-06-28 15:22:13 +0200] [27888] [INFO] Starting worker [27888]
INFO:sanic.root:Starting worker [27888]
[2021-06-28 15:22:18 +0200] - (sanic.access)[INFO][158.255.150.252:43194]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 15:23:02 +0200] - (sanic.access)[INFO][158.255.150.252:43204]: GET http://95.217.8.18:8000/login?username=user&password=pass  200 31
INFO:sanic.access:
[2021-06-28 15:34:16 +0200] - (sanic.access)[INFO][146.75.181.63:8076]: GET http://95.217.8.18:8000/login?username=user&password=pass  200 31
INFO:sanic.access:
[2021-06-28 15:34:52 +0200] - (sanic.access)[INFO][146.75.181.63:8137]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 15:34:55 +0200] - (sanic.access)[INFO][146.75.181.63:8151]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 16:01:31 +0200] - (sanic.access)[INFO][158.255.150.252:44212]: GET http://95.217.8.18:8000/?username=user&password=user  200 55
INFO:sanic.access:
[2021-06-28 16:02:12 +0200] - (sanic.access)[INFO][146.75.181.55:20680]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 16:02:16 +0200] - (sanic.access)[INFO][146.75.181.55:20687]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 16:02:37 +0200] - (sanic.access)[INFO][146.75.181.55:20724]: GET http://95.217.8.18:8000/complex/get  200 24
INFO:sanic.access:
[2021-06-28 16:02:38 +0200] [27888] [ERROR] Exception occurred while handling uri: 'http://95.217.8.18:8000/favicon.ico'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic_routing/router.py", line 84, in resolve
    extra,
  File "", line 14, in find_route
sanic_routing.exceptions.NotFound: Not Found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic/router.py", line 36, in _get
    extra={"host": host} if host else None,
  File "/usr/local/lib/python3.7/dist-packages/sanic_routing/router.py", line 96, in resolve
    raise self.exception(str(e), path=path)
sanic_routing.exceptions.NotFound: Not Found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic/app.py", line 726, in handle_request
    request.headers.getone("host", None),
  File "/usr/local/lib/python3.7/dist-packages/sanic/router.py", line 61, in get
    return self._get(path, method, host)
  File "/usr/local/lib/python3.7/dist-packages/sanic/router.py", line 39, in _get
    raise NotFound("Requested URL {} not found".format(e.path))
sanic.exceptions.NotFound: Requested URL /favicon.ico not found
ERROR:sanic.error:Exception occurred while handling uri: 'http://95.217.8.18:8000/favicon.ico'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic_routing/router.py", line 84, in resolve
    extra,
  File "", line 14, in find_route
sanic_routing.exceptions.NotFound: Not Found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic/router.py", line 36, in _get
    extra={"host": host} if host else None,
  File "/usr/local/lib/python3.7/dist-packages/sanic_routing/router.py", line 96, in resolve
    raise self.exception(str(e), path=path)
sanic_routing.exceptions.NotFound: Not Found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic/app.py", line 726, in handle_request
    request.headers.getone("host", None),
  File "/usr/local/lib/python3.7/dist-packages/sanic/router.py", line 61, in get
    return self._get(path, method, host)
  File "/usr/local/lib/python3.7/dist-packages/sanic/router.py", line 39, in _get
    raise NotFound("Requested URL {} not found".format(e.path))
sanic.exceptions.NotFound: Requested URL /favicon.ico not found
[2021-06-28 16:02:38 +0200] - (sanic.access)[INFO][146.75.181.55:20727]: GET http://95.217.8.18:8000/favicon.ico  404 673
INFO:sanic.access:
[2021-06-28 16:02:39 +0200] [27888] [ERROR] Exception occurred while handling uri: 'http:///*'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic/http.py", line 126, in http1
    await self.http1_request_header()
  File "/usr/local/lib/python3.7/dist-packages/sanic/http.py", line 188, in http1_request_header
    await self._receive_more()
  File "/usr/local/lib/python3.7/dist-packages/sanic/server.py", line 222, in receive_more
    await self._data_received.wait()
  File "/usr/lib/python3.7/asyncio/locks.py", line 293, in wait
    await fut
concurrent.futures._base.CancelledError
ERROR:sanic.error:Exception occurred while handling uri: 'http:///*'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sanic/http.py", line 126, in http1
    await self.http1_request_header()
  File "/usr/local/lib/python3.7/dist-packages/sanic/http.py", line 188, in http1_request_header
    await self._receive_more()
  File "/usr/local/lib/python3.7/dist-packages/sanic/server.py", line 222, in receive_more
    await self._data_received.wait()
  File "/usr/lib/python3.7/asyncio/locks.py", line 293, in wait
    await fut
concurrent.futures._base.CancelledError
[2021-06-28 16:02:39 +0200] - (sanic.access)[INFO][UNKNOWN]: NONE http:///*  503 666
INFO:sanic.access:
[2021-06-28 16:09:42 +0200] [27888] [INFO] Stopping worker [27888]
INFO:sanic.root:Stopping worker [27888]
[2021-06-28 16:09:43 +0200] [27888] [INFO] Server Stopped
INFO:sanic.root:Server Stopped
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 126
    except as e:
            ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 126
    except as e:
            ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 126
    except as e:
            ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from api import routes
  File "/root/leadersofdigital_splavy_rosseti/api/routes.py", line 126
    except as e:
            ^
SyntaxError: invalid syntax
[2021-06-28 16:37:11 +0200] [5698] [INFO] Goin' Fast @ http://95.217.8.18:8000
INFO:sanic.root:Goin' Fast @ http://95.217.8.18:8000
[2021-06-28 16:37:11 +0200] [5698] [INFO] Starting worker [5698]
INFO:sanic.root:Starting worker [5698]
[2021-06-28 16:39:15 +0200] - (sanic.access)[INFO][116.203.245.151:46206]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:39:32 +0200] - (sanic.access)[INFO][116.203.245.151:46272]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 16:39:36 +0200] - (sanic.access)[INFO][116.203.245.151:46272]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:39:46 +0200] - (sanic.access)[INFO][116.203.245.151:46352]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:39:56 +0200] - (sanic.access)[INFO][116.203.245.151:46418]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:40:06 +0200] - (sanic.access)[INFO][116.203.245.151:46438]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:40:18 +0200] - (sanic.access)[INFO][116.203.245.151:46456]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 16:40:20 +0200] - (sanic.access)[INFO][116.203.245.151:46456]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:53:03 +0200] - (sanic.access)[INFO][158.255.150.252:46048]: GET http://95.217.8.18:8000/login?username=admin&password=admin  200 45
INFO:sanic.access:
[2021-06-28 16:53:07 +0200] - (sanic.access)[INFO][158.255.150.252:46048]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 16:53:07 +0200] - (sanic.access)[INFO][158.255.150.252:46048]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
[2021-06-28 16:54:01 +0200] - (sanic.access)[INFO][176.116.137.101:33906]: GET http://95.217.8.18:8000/login?username=admin&password=pass  200 35
INFO:sanic.access:
[2021-06-28 16:54:02 +0200] - (sanic.access)[INFO][176.116.137.101:33906]: GET http://95.217.8.18:8000/complex/get  200 1203
INFO:sanic.access:
